[Linux] Cargo error with uucore

* **Platform** : Linux
* **Rust version** : `1.7.0`

When I try to build on linux, from scratch, I get the following error message from cargo:

```
cargo build  --features "base64 basename cat chmod chroot cksum comm cp cut dirname du echo env expand expr factor false fmt fold groups hashsum head hostid hostname id kill link ln logname mkdir mkfifo mktemp mv nice nl nohup nproc od paste printenv printf ptx pwd readlink realpath relpath rm rmdir seq shred shuf sleep sort split stdbuf sum sync tac tail tee test timeout touch tr true truncate tsort tty uname unexpand uniq unlink uptime users wc whoami yes"  --no-default-features
failed to parse manifest at `/home/antoine/rust_project/coreutils/src/uucore/Cargo.toml`

Caused by:
  expected a value of type `string` for the key `lib.name`
Makefile:237 : la recette pour la cible « build-uutils » a échouée
make: *** [build-uutils] Erreur 101

```

--- a/src/uucore/Cargo.toml
+++ b/src/uucore/Cargo.toml
@@ -10,3 +10,4 @@ winapi = "*"
 
 [lib]
 path = "lib.rs"
+name = "uucore"

